### ISSUE TITLE
META: Add Audio-Described Files to Video Component

### ISSUE THREAD

**Issue by andyvanavery31** [NCIOCPL/cgov-digital-platform#4318]
### Description
To meet accessibility requirements for video components, the user must be able to access the audio-described version of the video. This change request is to add these two items to the video component in a way that meets accessibility requirements, has easy navigation, and aligns with the look and feel of the latest redesign. This META ticket will track the related tickets to accomplish this across the platform.

### What's the expected change?
- The audio-described file for videos are added to the video content type and users can access them with videos.

### What's the current functionality?
- The audio-described file is not directly available to users on webpages

### Tracked Tickets

- [x] #4484 
- [x] #4487 
- [x] #4488 
- [ ] CTHPs Ticket

### Tasks

- [x] Need a designer to comp the front end of how to add the audio-described file to a video content type for:
  - [x] [Home & Landing Pages](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=8256-50052&t=MzAM3QUXBhFoEusF-1)
  - [x] [Mini Landing Pages](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=8256-50062&t=MzAM3QUXBhFoEusF-1)
  - [x] [Embedded Videos](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=8256-50059&t=MzAM3QUXBhFoEusF-1)
    - [x] @blilianyu Add comps of videos for each size (small, medium, large, and with and without captions) https://www-dev-ac.cancer.gov/api-test/a/embedded-video-test
  - [x] [CTHP Video Cards](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=8256-50056&t=MzAM3QUXBhFoEusF-1)
- [x] Need client review of designer solution for front-end view

### Resources
- [Figma](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=8256-50056&t=MzAM3QUXBhFoEusF-1)
- [Content Model](https://github.com/NCIOCPL/cgov-digital-platform/wiki/Media-Type-Video)

### Requirements

- The video component shall allow users to access the audio-described version of the video
- The audio-described version of the video will be accessed through an external link
- The audio-described version of the video will be optional for videos
- The addition of the audio-described file shall use a simple approach that does not change the video player itself
- The addition of the audio-described file shall assume that the video will no longer be a page-level content type
- If the video does not have an audio described version to go with it, then the designed interface to go to the audio-described version will not render
- The embedded video component update shall have the same user experience on the following embedding types on the Digital Platform:
  - Video Display: Large: No Title
  - Video Display: Large: Title
  - Video Display: Medium: No Title
  - Video Display: Medium: Title
  - Video Display: Small: No Title
  - Video Display: Small: Title
- The embedded video component can be placed on the Body Content WYSIWYG on the following Content Pages:
  - Application Page
  - Article Page
  - Biography
  - Blog Post
  - Cancer Center
  - Cancer Research List Page
  - Event
  - Press Release
- A link to videos can be added as a Related Resources Media Link on the following Content Pages:
  - Article Page
  - Biography
  - Blog Post
  - Cancer Center
  - Cancer Research List Page
  - Event
  - Press Release
- The video component appears in other forms on the following pages with the content type
  - Cancer Type Home Page - CTHP Video Card
  - Home & Landing - NCIDS Inline Video (NOT TO BE ADDRESSED IN THIS TICKET)
  - Mini Landing Page - 2 Column Layout (with NCIDS Style)
- The following blocks utilize videos, which will need to have the audio-described file available:
  - Content Block - the WYSIWYG
  - Video Carousel

Tickets to be made:

- Legacy Embedded in the WYSIWYG
  - Include the addition of the field to the CGDP Video Content Type to the first ticket being worked on
  - Update Full sized on Video Page
  - Make it appear on the front-end
  - With Caption and Without Caption, this needs to be within the caption.
  - If no caption, but an AD file, then the FigCaption should not be created, and "Audio Descriptions: Watch the audio-described version of this video" is not a description of the video
  - Will need to embed the combinations
- MLP
  -  Make it appear on the front-end
  - Follow NCIDS
  - If no caption, don't use FigCaption
- H&L
  -  Make it appear on the front-end
  - Follow NCIDS
- CTHP
  -  Make it appear on the front-end

- Need Spanish translation of the "Audio Descriptions: ",  "Watch the" "audio-described version of this video." Need to identify where the link appears

### Acceptance Criteria?
Front End Audio-Described File
```gherkin
Scenario: A visitor to a site page finds a video and wants to access the audio-described version
  Given the site visitor is on a page with a video
  When a sighted user sees the video
  Then the audio-described file will be visible
    And the user can access the file
    And the presentation will match the comp
  When a non-sighted user navigates through the page
  Then they will be able to navigate to the audio-described file
    And the user can access the file
```

Drupal Implementation
```gherkin
Scenario: A content editor is adding or editing a video content type on the CMS
  Given the user is logged into the CMS
  When the user is adding or editing a video content type on the CMS
  Then the fields of the video content type will match the content model (see link in Resources section)
```

**Comment by monika-jaeger** [comment]
Comps to review & discuss in Requirements Meeting on 9/25/24

https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=7085-33001&t=XlGlZSCC8Vl2Tjqd-1

cc: @blilianyu @andyvanavery31

**Comment by blilianyu** [comment]
@andyvanavery31 @monika-jaeger Here are the updated comps https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=7085-33001&t=YIF5ipaOSLb6BCcA-1
